src/main/scala/tech/cryptonomic/conseil/tezos/Tables.scala

```diff
@@ -1592,6 +1592,9 @@ trait Tables {
 
     /** Index over (level) (database name endorsing_rights_level_idx) */
     val index1 = index("endorsing_rights_level_idx", level)
+
+    /** Index over (delegate) (database name ix_delegate_slot) */
+    val index2 = index("ix_delegate_slot", delegate)
```

Comment:
The index reference here is not up-to-date, the Tables file needs be re-generated.
Alternatively remove the `_slot` appendix here and in the comment in the previous line.